docs/source/emb_sim.rst

```diff
@@ -0,0 +1,99 @@
+dglke_emb_sim: finding similar embeddings
+-------------------------------------------
+This task is to find the most similar entity/relation embeddings for some pre-defined similarity functions given a set of entities or relations. An example of the output for top5 similar entities are as follows::
+
+    left     right    score
+    0        0        0.99999
+    0        18470    0.91855
+    0        2105     0.89916
+    0        13605    0.83187
+    0        36762    0.76978
+
+Currently we support five different similarity functions: cosine, l2 distance, l1 distance, dot product and extended jaccard.
```

Comment:
i think we should define the similarity functions here.

Reply:
I added it under the argument description.